roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -11,6 +11,9 @@ data:
   # Operator specific settings
   EDA_DEPLOYMENT_TYPE: "k8s"
 
+  # Public Hostname
+  EDA_WEBHOOK_BASE_URL: http[s]://{{ public_hostname }}{{ eda_webhook_prefix_path }}
```

Comment:
@msmagnanijr When I wrote `http[s]` I meant that it can be `http` or `https` that's why I suggest to define `public_base_url` instead of `public_hostname`

Reply:
ahh got it! I thought that somehow this was already defined. let me update that.